### PR TITLE
array of colors

### PR DESCRIPTION
**Pull request by shulajz**
changed to array of color

### CODE REVIEW

**Comment by immAjzner** [comment]
from now please call branch names in the following way: name-of-branch and not nameOfBranch (camel case)

File: ex1/buttonActions.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
-function changeColor() {
 
-  if(document.getElementById("demo").style.backgroundColor == "blue")
-   {
-    document.getElementById("demo").style.backgroundColor = "red";
-  }
-  else {
-    document.getElementById("demo").style.backgroundColor = "blue";
-  }
+var i = 1;
```
Comment (immAjzner) [inline ex1/buttonActions.js:2]: please change to indexOfColor.  "i" has no meaning

File: ex1/buttonActions.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+function changeColor() {
+  var colorArray = ['#FF6633', '#CC9999','#4D80CC'];
+    //store
+    sessionStorage.setItem("color", colorArray[i]);
+    // Retrieve
+    document.getElementById("demo").style.backgroundColor = sessionStorage.getItem("color");
+    i++;
```
Comment (immAjzner) [inline ex1/buttonActions.js:9]: please remove this line and change the next line to:
... ++i  == colorArray.length ....

please read what ++ before a variable does in google

File: ex1/index.html [hunk 3]
```diff
@@ -12,7 +12,6 @@ <h1><u>The blue button</u></h1>
     <h2>This is my blue button</h2>
     <button type="button"
     class="button"
-    style="background-color:blue"
     onclick="changeColor()"
     id="demo">
```
Comment (immAjzner) [inline ex1/index.html:16]: I said to remove this id

File: ex1/buttonActions.js (hunk 2 above)
Comment (shulajz) [reply ex1/buttonActions.js:9]: i looked it up, we learned it in shana alef i think

**Pull request closed without merging**